corehq/apps/es/tests/test_case_search_adapter.py

```diff
@@ -89,3 +101,33 @@ def test_index_not_writes_to_bha_adapter_if_not_required(self):
 
         self.assertEqual(docs_in_bha, [])
         self.assertEqual(len(docs_in_case_search), 1)
+
+    def test_bulk_with_bha_mutliplexing(self):
+        with patch('corehq.apps.es.case_search.multiplex_to_adapter', return_value=case_search_bha_adapter):
+            case_search_adapter.bulk(self.bulk_index_actions, refresh=True)
+
+        # Cleanup
+        self.addCleanup(case_search_adapter.bulk, self.bulk_delete_actions, refresh=True)
+        self.addCleanup(case_search_bha_adapter.bulk, self.bulk_delete_actions, refresh=True)
+
+        docs_in_bha = self._get_normalized_cases_from_hits(case_search_bha_adapter.search({}))
+        docs_in_case_search = self._get_normalized_cases_from_hits(case_search_adapter.search({}))
+
+        self.assertEqual(docs_in_bha, docs_in_case_search)
```

Comment:
I think `self.assertCountEqual` is better to prevent a potential flaky test if results are returned in a different order between the two indices?

Reply:
This isn't a blocking comment and I don't plan to make any changes to the current PR related to this, just wanted to flag it.